save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/AvatarController.kt

```diff
@@ -86,7 +87,8 @@ internal class AvatarController(
             "Not found avatar for organization $organizationName with name $imageName"
         }
 
-    private fun AvatarKey.toMonoResponse(): Mono<ByteBufferFluxResponse> = avatarStorage.doesExist(this)
+    private fun AvatarKey.toMonoResponse(): Mono<ByteBufferFluxResponse> = this.toMono()
+        .filterWhen(avatarStorage::doesExist)
```

Comment:
Do we need an explicit `emptyToNotFound` here?

Reply:
fixed